src/main/scala/org/zalando/spearheads/innkeeper/services/RouteToEskipTransformer.scala

```diff
@@ -57,7 +57,9 @@ class DefaultRouteToEskipTransformer @Inject() (envConfig: EnvConfig, hostsServi
 
   private[this] def createHostPredicate(hostIds: Seq[Long]) = {
     val hosts = hostsService.getByIds(hostIds.toSet)
-    val hostsString = hosts.mkString("|")
+    val hostsString = hosts
+      .map(_.replace(".", "[.]"))
+      .mkString("|")
     val hostsRegex = s"/^$hostsString$$/"
```

Comment:
This should be `val hostsRegex = s"/^($hostsString)$$/"` to be consistent with our internal eskip files